### DISCUSSION TITLE
convert_unix_time

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#76] [Q&A]
Didn't you forget the leap second :-) ?
https://en.wikipedia.org/wiki/Leap_second
 
Is BleskOS limited to 32 bit time ending in 2038 ?

Could be usefull to have two functions to convert unix time:
- one quicker for present year.
- one slow with each leap second for past years.

**Comment by VendelinSlezak** [comment]
Yes, I did not even know about it :) I will fix it.
